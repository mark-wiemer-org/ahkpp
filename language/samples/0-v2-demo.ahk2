#HotIf WinActive("ahk_class Notepad") or WinActive(MyWindowTitle)
#Space::MsgBox "You pressed Win+Spacebar in Notepad or " MyWindowTitle
#HotIf
