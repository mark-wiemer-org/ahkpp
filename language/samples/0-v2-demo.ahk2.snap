>#HotIf WinActive("ahk_class Notepad") or WinActive(MyWindowTitle)
#^ source.ahk2 keyword.control.directive.ahk2 punctuation.definition.directive.ahk2
# ^^^^^ source.ahk2 keyword.control.directive.ahk2
#      ^ source.ahk2
#       ^^^^^^^^^ source.ahk2 entity.name.function.ahk2 support.function.ahk2
#                ^ source.ahk2 meta.parens.ahk2 punctuation.section.parens.begin.bracket.round.ahk2
#                 ^ source.ahk2 meta.parens.ahk2 string.quoted.ahk2 punctuation.definition.string.begin.ahk2
#                  ^^^^^^^^^^^^^^^^^ source.ahk2 meta.parens.ahk2 string.quoted.ahk2
#                                   ^ source.ahk2 meta.parens.ahk2 string.quoted.ahk2 punctuation.definition.string.end.ahk2
#                                    ^ source.ahk2 meta.parens.ahk2 punctuation.section.parens.end.bracket.round.ahk2
#                                     ^ source.ahk2
#                                      ^^ source.ahk2 keyword.operator.expression.ahk2
#                                        ^ source.ahk2
#                                         ^^^^^^^^^ source.ahk2 entity.name.function.ahk2 support.function.ahk2
#                                                  ^ source.ahk2 meta.parens.ahk2 punctuation.section.parens.begin.bracket.round.ahk2
#                                                   ^^^^^^^^^^^^^ source.ahk2 meta.parens.ahk2 variable.other.ahk2
#                                                                ^ source.ahk2 meta.parens.ahk2 punctuation.section.parens.end.bracket.round.ahk2
>#Space::MsgBox "You pressed Win+Spacebar in Notepad or " MyWindowTitle
#^^^^^^ source.ahk2 hotkey.ahk2 keyword.keys.ahk2
#      ^^ source.ahk2 punctuation.definition.colon
#        ^^^^^^ source.ahk2 variable.other.ahk2 support.function.ahk2
#              ^ source.ahk2
#               ^ source.ahk2 string.quoted.ahk2 punctuation.definition.string.begin.ahk2
#                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.ahk2 string.quoted.ahk2
#                                                       ^ source.ahk2 string.quoted.ahk2 punctuation.definition.string.end.ahk2
#                                                        ^ source.ahk2
#                                                         ^^^^^^^^^^^^^ source.ahk2 variable.other.ahk2
>#HotIf
#^ source.ahk2 keyword.control.directive.ahk2 punctuation.definition.directive.ahk2
# ^^^^^ source.ahk2 keyword.control.directive.ahk2
>